trackers/node-tracker/docs/node-tracker.api.md

```diff
@@ -360,6 +360,7 @@ export type Timestamp = TrueTimestamp | DeviceTimestamp | number;
 export interface Tracker extends TrackerCore {
     setDomainUserId: (userId: string) => void;
     setNetworkUserId: (userId: string) => void;
+    setSessionId: (sessionId: string) => void;
```

Comment:
Nit: Session ID and the session index (`vid` property in the payload, `domain_sessionidx` in the atomic events) are quite related. Could we also add that option here? It's not super important, we have other trackers where only session ID is possible to set and not the index, but might be better to have both done in one go.

Reply:
Done!